src/kontrol/prove.py

```diff
@@ -480,6 +480,10 @@ def _method_to_cfg(
         init_node = cfg.create_node(init_cterm)
         new_node_ids = [init_node.id]
         init_node_id = init_node.id
+        if not method.is_testfail and not method.is_test:
+            _LOGGER.warning(
+                f'{contract.name_with_path}.{method.signature} is not prefixed with test or testFail, therefore it is not being checked for failures.'
```

Comment:
I have two suggestions here: 
1. This happens if a function is not prefixed with `test`, `check`, or `prove`
2. I'm now not sure if it "not being checked for failures" is clear to the user (I understand it's a formulation from an issue, but I'd try to refine it a bit). Maybe we can say
- "`foundry` or `hevm` success predicate is not checked" (that's technically correct, but may not be clear to external users)
- "reverts and assertion violations are not reported"
- "it is not reported as failing in presence of reverts or assertion violations"

Or something along these lines.